config/config.go

```diff
@@ -16,6 +21,13 @@ type Config struct {
 	SQS       SQS       `toml:"sqs"`
 	API       API       `toml:"api"`
 	Transport Transport `toml:"transport"`
+
+	// DataTransportConfig specifies config for the data transporter.
+	// It will be one of the following types:
+	// - *MemoryTransportConfig
+	// - *QueueTransportConfig
+	// - *LocalTransportConfig
+	DataTransport DataTransportConfig `toml:"dataTransport"`
```

Comment:
Let's use the existing `Transport` struct. It isn't being used right now anyway.